### DISCUSSION TITLE
what is "blocked" field for ?

### DISCUSSION THREAD

**Discussion by sulthanalihsan** [serverpod/serverpod#1056] [Q&A] [answered]
![image](https://github.com/serverpod/serverpod/assets/30169687/efbdc188-1332-4c1f-a8ad-91394f016a8b)

i'm wondering what is "blocked" field in serverpod_user_info table for?
is it working or not for now? and how do I update it through Users method?

![image](https://github.com/serverpod/serverpod/assets/30169687/eeb9b872-3eb3-43a9-8ac5-401618346df3)

thanks

**Comment by vlidholt** [comment, marked as answer]
It's not really used anywhere, but it can be convenient to be able to set manually to keep track on users that are blocked from logging in. Adding this as an automatic change may need to wait until v 2 as it would be a breaking change.

**Comment by OsvaldoTCF** [comment]
How do I block a user in ServerPod 3?

Is there a class or method within a class that does this?

**Reply by SandPod** [reply]
AuthUser still has the blocked property. It now prevents users from creating authentication tokens with the server. So if you sign out a user and want to keep it signed out, you can make the user blocked.

To update the users status, use `AuthServices.instance.authUsers.update(...);`
